docs/examples/2Dwavefunc_animated.py

```diff
@@ -0,0 +1,184 @@
+"""
+===============================================
+Animating a time-varying 2D wave function
+===============================================
+
+This is a simple demonstration of how one can visualize
+time-varying 2D wave functions using FURY.
+
+Some other interesting functions you can try plotting -
+#z = (x**2 - y**2)/(x**2 + y**2)**0.5*np.cos(np.pi*x-rate*time)
+#z = np.sin(x**2)*np.cos(y**2-rate*incre)#*np.sin(rate*time)
+#z = np.sin(x**2-y**2)*np.sin(time)
```

Comment:
Additionally, if multiple functions are showcased please adjust the label corresponding to each function.